docs/agent/sensor-full-disk-access.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[sensor-full-disk-access]]
+== Enable full disk access
+
+Elastic cloud users can take advantage of Endgame's detection sensor. However, after installing the sensor on either macOS 10.14 Mojave or 10.15 Catalina, you must enable a setting called **Full Disk
+Access** so the sensor can analyze files that are in your home directory.
```

Comment:
Is there a reason why we focus on Elastic Cloud Users? 

Suggested edits:
macOS 10.14 and greater requires users to provide authorization to access the home directory. In order to ensure you are fully protected, Elastic Endpoint Security requires full disk access. Please follow the instructions below to enable full disk access.